src/99-appendices/03-hed.md

```diff
@@ -75,5 +75,42 @@ The tags in the `HED` column are often specific to the event instances, while
 the common properties associated with categories such as `mycodes` are
 encapsulated in the \_events.json dictionary. Downstream tools should not
 distinguish between tags specified using the different mechanisms. Further,
-the normal BIDS inheritance principle applies so these data dictionaries can
+the normal BIDS inheritance principle applies, so these data dictionaries can
```

Comment:
I suggest to add a link to the BIDS inheritance principle here, and remove the word `"normal"`

```suggestion
the [BIDS inheritance principle](../02-common-principles.md#the-inheritance-principle) applies,
so these data dictionaries can
```